### PR TITLE
Fix legacy routes like ajax.php on nginx

### PR DESCRIPTION
**Pull request by ChristophWurst**
Suggestions from @jlehtoranta, taken from https://github.com/nextcloud/documentation/pull/2197#issuecomment-721432337

Without this you can't use the LDAP wizard. With the changes it works.

### CODE REVIEW

File: admin_manual/installation/nginx-subdir.conf.sample [hunk 1]
```diff
@@ -113,6 +113,9 @@ server {
         # then Nginx will encounter an infinite rewriting loop when it prepends
         # `/nextcloud/index.php` to the URI, resulting in a HTTP 500 error response.
         location ~ \.php(?:$|/) {
+            # Required for legacy support
+            rewrite ^/(?!index|remote|public|cron|core\/ajax\/update|status|ocs\/v[12]|updater\/.+|oc[ms]-provider\/.+|.+\/richdocumentscode\/proxy) /index.php$request_uri;
```
Comment (kesselb) [inline admin_manual/installation/nginx-subdir.conf.sample:117]:
```suggestion
            rewrite ^/(?!index|remote|public|cron|core\/ajax\/update|status|ocs\/v[12]|updater\/.+|oc[ms]-provider\/.+|.+\/richdocumentscode\/proxy) /nextcloud/index.php$request_uri;
```

Maybe?

**Review by small1 — APPROVED** [review]

**Pull request merged**